data/templates/install-photon/photon-os-ks

```diff
@@ -29,14 +29,14 @@
                         <%_ numSize = 0; _%>
                     <%_ } _%>
                     <%_ if (par.fsType === "swap") { _%>
-                        <%_ if (i === length ) { _%>
-        {"size": <%=numSize%>, "filesystem": "<%=par.fsType%>"}
-                        <%_ } else { _%>
-        {"size": <%=numSize%>, "filesystem": "<%=par.fsType%>"},
-                        <%_ } _%>
+                        <%_ swapPar = par; _%>
+                        <%_ swapParSize = numSize; _%>
                     <%_ } else { _%>
-                        <%_ if (i === length ) { _%>
+                        <%_ if (i === length && typeof swapPar === 'undefined') { _%>
         {"mountpoint": "<%=par.mountPoint%>", "size": <%=numSize%>, "filesystem": "<%=par.fsType%>"}
+                        <%_ } else if (i === length && typeof swapPar !== 'undefined') { _%>
+        {"mountpoint": "<%=par.mountPoint%>", "size": <%=numSize%>, "filesystem": "<%=par.fsType%>"},
```

Comment:
it's just a rendered output in ks file for os installer, not the logic itself when rendering. i think it more readable when putting it in one line, so the reader would know that in a certain condition, something will be rendered, and they just need to skip one line to check other conditions.